stdlib/CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ option(SWIFT_ENABLE_COMPATIBILITY_OVERRIDES
   "Support back-deploying compatibility fixes for newer apps running on older runtimes."
   TRUE)
 
+option(SWIFT_STDLIB_USE_NONATOMIC_RC
+    "Build the standard libraries and overlays with nonatomic reference count operations enabled"
+    FALSE)
+
+option(SWIFT_STDLIB_SINGLETHREADED_RUNTIME
```

Comment:
WDYT of spelling this `SWIFT_STDLIB_SINGLE_THREADED_RUNTIME`?  It makes it easier to read IMO.

Reply:
Yup. Switched to that new spelling.